### PR TITLE
add pt-br/horizontal-scaling

### PR DESCRIPTION
**Pull request by rafaelmnatali**
### Describe your changes
Added definition for `horizontal scaling`

### Related issue number or link (ex: `resolves #issue-number`)
https://github.com/cncf/glossary/issues/824

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/pt-br/horizontal-scaling.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+category: Conceito
+tags: ["infrastructure", "", ""]
+---
+
+Escalabilidade horizontal é uma técnica em que a capacidade de um sistema é aumentada adicionando mais [nós](/pt-br/nodes/)
+em vez de adicionar mais recursos de computação a nós individuais (sendo este último conhecido como [escalonamento vertical](/pt-br/vertical-scaling/)).
+Vamos supor que temos um sistema com 4GB de RAM e queremos aumentar sua capacidade para 16GB de RAM.
+Escalá-lo horizontalmente significa fazer isso adicionando 4 x 4GB de RAM em vez de trocar para um sistema de 16GB de RAM.
+
+Essa abordagem melhora o desempenho de uma aplicação adicionando novas instâncias, ou n ós, para distribuir melhor a carga de trabalho.
```
Comment (edsoncelio) [inline content/pt-br/horizontal-scaling.md:13]: nit:"n ós"

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Thanks! Just a tiny typo: after that, it will be LGTM!

File: content/pt-br/horizontal-scaling.md (hunk 1 above)
Comment (rafaelmnatali) [reply content/pt-br/horizontal-scaling.md]: fixed.

**Review by edsoncelio — APPROVED** [review]
LGTM
Thanks!!

**Comment by edsoncelio** [comment]
PTAL @brunoguidone @jessicalins

File: content/pt-br/horizontal-scaling.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: Escalonamento Horizontal
+status: Completed
+category: Conceito
+tags: ["infrastructure", "", ""]
```
Comment (MrErlison) [inline content/pt-br/horizontal-scaling.md:5]: You forgot the tag.

Comment (MrErlison) [reply content/pt-br/horizontal-scaling.md:5]: "infraestrutura" instead "infrastructure"

Like Edson said, "just a tiny fix". After that LGTM!

Thank you!

**Pull request closed without merging**